### ISSUE TITLE
fakeroot not installing due to outdated reference to 1.24

### ISSUE THREAD

**Issue by krombel** [Homebrew/homebrew-core#72159]
**`brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output**

% brew gist-logs fakeroot
Error: No logs.

****

- [X] The `brew doctor` above contains no "Warning" lines.

**What were you trying to do (and why)?**

Install fakeroot as it is a dependency of a project

**What happened (include all command output)?**
```
% brew install --build-from-source fakeroot
==> Downloading https://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;filename=0001-Implement-openat-2-wrapper-which-handles-optional-
######################################################################## 100.0%
==> Downloading https://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;filename=0002-OS-X-10.10-introduced-id_t-int-in-gs-etpriority.pa
######################################################################## 100.0%
==> Downloading https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.24.orig.tar.gz
##O#- #                                                                       
curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "fakeroot"
Download failed: https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.24.orig.tar.gz
```
**What did you expect to happen?**

it installs fakeroot

**Step-by-step reproduction instructions (by running `brew` commands)**

brew install --build-from-source fakeroot

**Comment by carlocab** [comment]
Sorry, but source builds are unsupported, and there's nothing we can do if upstream takes a tarball out of commission.

You can try either installing from a bottle or opening a PR to update the formula.

This previous attempt might be informative: #64906

**Comment by krombel** [comment]
Ohh. I didn't see that PR. I searched for open issues for fakeroot but missed that there might have been a PR to update that already.
Sorry for the noise then. I am sadly not familiar (enough) with homebrew to fix my issue (summed up in #65000) myself...
